crates/workspace/src/pane_group.rs

```diff
@@ -720,27 +723,49 @@ impl PaneAxis {
         render_cx: &dyn PaneLeaderDecorator,
         window: &mut Window,
         cx: &mut App,
-    ) -> gpui::AnyElement {
+    ) -> (gpui::AnyElement, bool) {
         debug_assert!(self.members.len() == self.flexes.lock().len());
         let mut active_pane_ix = None;
+        let mut contains_active_pane = false;
+
+        // Track which children are leaf panes (Member::Pane) vs axes (Member::Axis)
```

Comment:
That seems like a good `///` comment on an actual struct field, let's move it there.